Wrappers/Python/cil/recon/FBP.py

```diff
@@ -294,17 +292,21 @@ class FDK(GenericFilteredBackProjection):
 
     Example
     -------
+    >>> from cil.utilities.dataexample import SIMULATED_CONE_BEAM_DATA
+    >>> from cil.recon import FDK
+    >>> data = SIMULATED_CONE_BEAM_DATA.get()
     >>> fdk = FDK(data)
     >>> out = fdk.run()
 
     Notes
     -----
     The reconstructor can be futher customised using additional 'set' methods provided.
     """
-    
+    supported_backends = ['tigre']
```

Comment:
Do you need to have this? Is it not inherited from the base class?

Reply:
It doesn't harm to specify as it will then be clearer when reading the class. I'm just wondering if this would be available or not.

Reply:
Yes, I believe it's inherited, I just thought it would add clarity to specify here. In the future we might change the baseclass defaults.